packages/terra-card/src/_themes.scss

```diff
@@ -0,0 +1,4 @@
+@include cerner-consumer-theme {
+  --terra-card-box-shadow: 0 1px 3px 0 rgba(0, 0, 0, 0.3);
```

Comment:
These are not the correct values for the box-shadow, they should be:
`--terra-card-box-shadow: 0 1px 1px 0 rgba(28,31,33, 0.35);`

Reply:
Where did you get that value? I'm looking in callouts and although there are many box-shadow styles, that one I can't find.... 

We are uplifting to cards in HealtheLife right now and `0 1px 3px 0 rgba(0,0,0,0.3)` is what is in use. I don't particularly care either way.I just want consistency.

Reply:
@mhemesath - responding a little late: 

The new OCS:Consumer callouts for Cards were originally posted on their wiki here:
https://wiki.ucern.com/pages/viewpage.action?pageId=1627595411

which I transferred to the finalized callouts on the Terra Approved Patterns wiki (both in the PDF and on the third tab labeled "cerner-consumer-theme"): https://wiki.ucern.com/x/Bn4YYQ

Going forward, we will be consolidating all callouts either to Zeplin or the Terra Approved Patterns wiki so this will be easier to see all theme callouts for each component in one location. I checked with Chris and the box-shadow being uplifted in HealtheLife right now is slightly different from what is being designed in the new OCS:Consumer theme for Terra.